databases/gcp-mysql-ssl/README.md

```diff
@@ -0,0 +1,32 @@
+# Heroku PgBouncer Buildpack
+
+Google Cloud MySQL with SSL only connection. Requires
+
+Prisma docs: https://www.prisma.io/docs/concepts/database-connectors/mysql#configuring-an-ssl-connection
+GCP docs: https://cloud.google.com/sql/docs/mysql/configure-ssl-instance
+
+## How to run this locally
+
+### Environment variables
+
+The environment variable `GCP_MYSQL_SSL_DB_URL` should point to a GCP MySQL database. Download the certificates based on GCP docs and put them in the root directory next to this `README.md` file.
```

Comment:
can you point to the file from the repo here? just point out the file name and if you needed to do anything besides downloading

Reply:
The file can't be in the repo because they are secrets. In CI, they are in env vars and are written to the filesystem dynamically.

Reply:
The specific file names are listed below in the readme

Reply:
Suggest a reorder if you prefer or if you mean something else.